ha/const.py

```diff
@@ -95,6 +95,8 @@
 # Systemd wrapper resource agent
 HARE_FID_MAPPING_FILE="/var/lib/hare/consul-server-conf/consul-server-conf.json"
 
+# Send IEC to syslog
+IEC="logger -i -p local3.err"
```

Comment:
what is "local3.err" for here?

Reply:
We are using **logger** command line tool to dump a log to syslog. -i is responsible for logging the PID with the message.
-p specifies the priority of the process. priority is given in the form of **facility.level**. So, local3 is the facility. facility signifies which process of the machine has created the message. So, it will log error message in the local3 facility.
local3 facility messages are for **network news subsystem**. 
I just got this command and I am using it directly. Please suggest me if any change is required.

Reply:
Please check with other components once e.g. s3server regarding what facility they use.

Reply:
Ok.

Reply:
Refering to the LLD for R2 and also checked with Ajay.
https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/278627397/Alert+Generator
Its using local3 facility only.